daliuge-translator/dlg/dropmake/web/main.js

```diff
@@ -202,14 +202,16 @@ function updateDeployOptionsDropdown() {
 async function checkUrlStatus(url) {
     return new Promise((resolve) => {
         $.ajax({
-            url: url,
-            type: 'HEAD',
-            dataType: 'jsonp',
+            url: url + 'api',
+            method: 'HEAD',
+            // dataType: 'jsonp',
```

Comment:
Is this incorrectly commented out? Otherwise, we can delete it.

Reply:
Yes, we can remove the commented line. I'll do that.